tests/test_data/weightwatchers.com/weightwatcherspublic_1.json

```diff
@@ -0,0 +1,50 @@
+{
+  "author": "WeightWatchers",
+  "canonical_url": "https://www.weightwatchers.com/de/rezept/kartoffelgulasch/562a9b02873e1afb2a3c4c13",
+  "site_name": "WW Deutschland",
+  "host": "weightwatchers.com",
+  "language": "en",
```

Comment:
Kannst wir Automatisch Sprachauswahl hier?

Reply:
Aus irgendeinem Grund kann ich nicht.

I'm wondering where is is gathering this "en" from to begin with. I haven't been able to track it down

Reply:
Possibly the `lang` tag(s) on the `<html>` element?  It seems to be duplicated there -- both `lang=de` and `lang=en`.

Reply:
it can be gathered from the `xml:lang` attribute from the `<html>` tag. That seems like the best way to do it without any manual overrise

Reply:
Sounds good.  Thank you!

Reply:
I realized later that this bugreport was off-topic for the pull request.  I've filed #1215 for that.

I think it's OK to resolve both bugs with this pull request if you like -- strictly speaking one per bug would be ideal I think, but I'm OK with either approach here (the diff is small).